src/commands/add/addCredentialCommand.ts

```diff
@@ -0,0 +1,55 @@
+import { TargetScope } from '../../types'
+import { CommandExample, ReturnCode } from '../../types/command'
+import { TargetCommand } from '../../types/command/targetCommand'
+import { addCredential } from './addCredential'
+
+// The syntax here needs to use square brackets for the subCommand(i.e. make it optional) since
+// the alias `auth` does not have a subCommand
+const syntax = 'add [subCommand]'
```

Comment:
We repeat the same logic to create `logic`, `aliases`, `usage`, `description` and examples, so it would be nice to create a utility for it